insight/tasks/user_weekly_trend_analysis.py

```
@@ -0,0 +1,185 @@
+from weekly_llm_analyzer import analyze_user_posts
+
+from insight.models import UserWeeklyTrend
+from posts.models import Post, PostDailyStatistics
+from scraping.velog.client import VelogClient
+from users.models import User
+from utils.utils import get_previous_week_range
+
+logger = logging.getLogger("scraping")
+
+
+async def run_weekly_user_trend_analysis(user, velog_client, week_start, week_end):
```

Pylint (3.3.7)

[refactor] Too many local variables (21/15)

(R0914)